src/Shared/Components/SelectPicker/type.ts

```diff
@@ -84,6 +85,11 @@ declare module 'react-select/base' {
          * @default 'true'
          */
         showSelectedOptionIcon?: boolean
+        /** Renders "Load more" button at the end of the menu list. */
+        loadMoreButtonConfig?: Pick<ButtonProps, 'isLoading' | 'disabled'> & {
+            show?: boolean
+            onClick?: () => void
+        }
```

Comment:
We can make this generic instead. Like adding a renderOptionsFooter instead of hard coding to button. Had a discussion with Vivek as well regarding the same and he seems to be onboard with it. Pls discuss